lib/game_stats.rb

```diff
@@ -1,28 +1,19 @@
-require_relative 'game'
-require_relative 'team_stats'
-require_relative 'data_loadable'
-
 class GameStats
-  include DataLoadable
   attr_reader :games
+  def initialize(games)
+    @games = games
+  end
 
-  def initialize(file_path, object)
-    @games = csv_data(file_path, object)
+  def highest_total_score
+    @games.map { |game| game.away_goals + game.home_goals }.max
   end
 
-  def percentage_ties
-    ties = @games.count do |game|
-      game.away_goals == game.home_goals
-    end
-    ties.fdiv(@games.length).round(2)
+  def lowest_total_score
+    @games.map { |game| game.away_goals + game.home_goals }.min
   end
 
-  def count_of_games_by_season
-    #sort by number/games_by_season
-    @games.reduce(Hash.new(0)) do |games_by_season, game|
-      games_by_season[game.season] += 1
-      games_by_season
-    end
+  def biggest_blowout
+    @games.map { |game| (game.away_goals - game.home_goals).abs }.max
```

Comment:
I'm guessing the idea is to format as the interaction pattern calls the methods?